### ISSUE TITLE
Disabling cutout augmentation doesn't seem to have much of an effect

### ISSUE THREAD

**Issue by tannisroot** [lucidrains/stylegan2-pytorch#147]
Due to my training results being quite negatively distorted by cutout/cropping augmentation type, I decided to try training with only translation augmentation type (using "--aug-types [translation]` parameter). However, even with that, I still get similar "cropped" results in training previews, as if disabling cutout had no effect whatsoever.
Is it possible that cutout augmentation doesn't get disabled even with that option? Or is it possible the network is doing that all by itself for some reason?

**Comment by lucidrains** [comment]
@tannisroot It could be the translation as well, I think it's best to try just the color augmentation alone and see if that helps. That's really weird, because Karras' new paper suggests that a low enough augmentation probability should prevent that from happening. But I see it as well in my own training

**Comment by lucidrains** [comment]
@tannisroot of course, if you have enough data, you can disable it altogether!

**Comment by tannisroot** [comment]
Oh, translation might cause this? Then what does it do exactly (same question for cutout). I always though it just did horizontal flip or something like that.
And unfortunately I only have 1400 images in my dataset, and from what I've seen augmentation is quite needed for such small sets.

**Comment by lucidrains** [comment]
@tannisroot yes, 1400 is indeed too small a dataset for GANs without augmentation

Feel free to try this new alternative technique though! https://github.com/lucidrains/denoising-diffusion-pytorch
